packages/lending/test/transactions/collateral-swap.test.ts

```diff
@@ -32,132 +34,130 @@ describe('Transaction: Collateral swap', function () {
       {
         protocolId: 'aave-v2',
         marketId: 'mainnet',
-        account: '0x7F67F6A09bcb2159b094B64B4acc53D5193AEa2E',
-        srcToken: mainnetTokens.WBTC,
-        srcAmount: '1',
-        srcAToken: aaveV2.mainnetTokens.aWBTC,
-        destToken: mainnetTokens.WETH,
-        destAToken: aaveV2.mainnetTokens.aWETH,
-        expects: {
-          logicLength: 7,
-        },
+        srcToken: mainnetTokens.WETH,
+        srcAmount: initSupplyAmount,
+        srcAccountingToken: aaveV2.mainnetTokens.aWETH,
+        destToken: mainnetTokens.WBTC,
+        destAccountingToken: aaveV2.mainnetTokens.aWBTC,
+        expects: { logicLength: 7 },
```

Comment:
initSupplyAmount is 2, so the final logics will have `Add Funds: 2 aWETH`. I think the user's address doesn't have enough aWETH to pass the test.

Reply:
It's okay, my mistake, just see the part about the deposit

Reply:
The `initSupplyAmount` is 2 and the `collateralSwapAmount` is 2 which seems no problem (and the test was passed). But for consistency, I will change the amount to 1 as other test cases.